Lib/test/test_c_locale_coercion.py

```diff
@@ -15,27 +15,36 @@
     interpreter_requires_environment,
 )
 
+# Set the list of ways we expect to be able to ask for the "C" locale
+EXPECTED_C_LOCALE_EQUIVALENTS = ["C", "invalid.ascii"]
```

Comment:
I think `""` should be included in this list otherwise the case where all the locale envt variables are set to `""` may not be tested. On Android it is only the tests that have `coerce_c_locale="warn"` that must be skipped when all locale envt variables are set to `""`.

There is currently no test run for when none of these variables is defined and AFAIK it seems correct since it is equivalent to having all locale envt variables set to `""`. But since it is such a common case, testing with all locale envt variables set to `""` should be mandatory for all platforms.

Maybe `EXPECTED_C_LOCALE_EQUIVALENTS` could be replaced with `LOCALES_TO_TEST`.

Reply:
That makes sense. If we go that way, then what I'd actually suggest we do is give the empty locale a dedicated test case, separate from the C locale equivalents.

Reply:
And a dedicated test case for the empty locale would be more explicit.

Reply:
+1 to a dedicated test case for empty locale.